src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,92 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "cinema_hall_id")
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
+
+    public MovieSession() {
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public Movie getMovie() {
+        return movie;
+    }
+
+    public void setMovie(Movie movie) {
+        this.movie = movie;
+    }
+
+    public CinemaHall getCinemaHall() {
+        return cinemaHall;
+    }
+
+    public void setCinemaHall(CinemaHall cinemaHall) {
+        this.cinemaHall = cinemaHall;
+    }
+
+    public LocalDateTime getShowTime() {
+        return showTime;
+    }
+
+    public void setShowTime(LocalDateTime showTime) {
+        this.showTime = showTime;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        MovieSession that = (MovieSession) o;
+        return Objects.equals(id, that.id) && Objects.equals(movie, that.movie)
+                && Objects.equals(cinemaHall, that.cinemaHall)
+                && Objects.equals(showTime, that.showTime);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, movie, cinemaHall, showTime);
+    }
+
+    @Override
+    public String toString() {
+        final StringBuilder sb = new StringBuilder("MovieSession{");
+        sb.append("id=").append(id);
+        sb.append(", movie=").append(movie);
+        sb.append(", cinemaHall=").append(cinemaHall);
+        sb.append(", showTime=").append(showTime);
+        sb.append('}');
+        return sb.toString();
+    }
```

Comment:
for both your models: let's use simple string concatenation for `toString()`
under the hood it uses StringBuilder in this case